# 0xFB : Status Message - BIT Health

# Variables declaration
	uint8 SyncByte  # IMU Address
	uint8 MessageID  # Message ID
	uint16 crc_fail  # CRC failure flags
	uint16 hardware_fail  # Hardware failure flags
	uint16 Checksum  # Uint16 Checksum. Complement = False
